optimism-children-indexer/optimism-children-indexer-proto/src/lib.rs

```diff
@@ -22,7 +22,7 @@ impl From<PaginationMetadata> for v1::Pagination {
     fn from(value: PaginationMetadata) -> Self {
         Self {
             page: value.pagination.page,
```

Comment:
```suggestion
            page: value.pagination.page + 1,
```

Ref https://github.com/Golem-Base/blockscout-frontend/blob/77b768000a55075da989f04344d91e18b8c7f5d3/lib/api/services/paginationConverter.ts#L24